controllers/tasks.js

```diff
@@ -16,6 +16,7 @@ const addNewTask = async (req, res) => {
     const body = {
       ...req.body,
       createdBy,
+      taskNo: await tasks.generateTaskNo(),
```

Comment:
shouldn't this be like `taskNum: ` and where are we handling error?

Reply:
done, good find